processor/ledger_for_evaluator.go

```diff
@@ -0,0 +1,140 @@
+package processor
+
+import (
+	"fmt"
+
+	"github.com/algorand/go-algorand/data/basics"
+	"github.com/algorand/go-algorand/data/bookkeeping"
+	"github.com/algorand/go-algorand/ledger"
+	"github.com/algorand/go-algorand/ledger/ledgercore"
+)
+
+// LedgerForEvaluator implements the indexerLedgerForEval interface from
```

Comment:
Skipping over this file for now. Hopefully we can use existing go-algorand code to handle fetching the inputs.